### PR TITLE
Add include to command.php so WP-CLI commands are registered

### PR DESCRIPTION
**Pull request by caseypatrickdriscoll**
https://github.com/scribu/wp-posts-to-posts/issues/504

### CODE REVIEW

**Comment by caseypatrickdriscoll** [comment]
This solution doesn't work.

Fatal error: Class 'WP_CLI' not found in /srv/www/prism/htdocs/wp-content/plugins/posts-to-posts/command.php on line 3

Not sure how best to do this, will think it over.

File: posts-to-posts.php [hunk 1]
```diff
@@ -10,6 +10,8 @@
 Domain Path: /lang
 */
 
+include 'command.php';
```
Comment (danielbachhuber) [inline posts-to-posts.php]: Change this to:

```
if ( defined( 'WP_CLI' ) && WP_CLI ) {
    require_once dirname( __FILE__ ) . '/command.php';
}
```

**Comment by danielbachhuber** [comment]
You can also use `command.php` without the code change by doing:

```
wp --require=command.php p2p
```

**Comment by caseypatrickdriscoll** [comment]
Yeah that might be better. If I get the time I'll update the docs.

I added a class_exists check that works, but I'll leave it @scribu

**Pull request merged**